### DISCUSSION TITLE
Attach WkWebView layer to MediaMixer().

### DISCUSSION THREAD

**Discussion by VivekVardhanXcel** [HaishinKit/HaishinKit.swift#1638] [Q&A]
Hi @shogo4405.

How can I attach WkWebView layer to MediaMixer().

Can you help me with this?

**Comment by shogo4405** [comment]
I haven't tried it before, so I'm not sure. Please give it a try and let me know how it's done.

**Comment by shogo4405** [comment]
close as duplicate https://github.com/shogo4405/HaishinKit.swift/discussions/1480
